tekton/ci-workspace/jobs/tekton-golang-tests.yaml

```diff
@@ -6,7 +6,8 @@ metadata:
 spec:
   iterateParam: context
   taskRef:
-    name: golang-test  # from the catalog
+    name: golang-test
+    bundle: gcr.io/tekton-releases/catalog/upstream/golang-test:0.1
```

Comment:
I'm not 100% sure this will work.
I hope that the taskloop controller will just copy the `taskRef` to the `TaskRun` and not attempt to resolve it.

Reply:
I tested this, but it runs, but only as long as the task is also installed in the cluster.
The custom task controller does not try to fetch the task, but it only passes the name from the task ref to the taskrun, not the bundle.

Reply:
Fix for the task loop controller: https://github.com/tektoncd/experimental/pull/738